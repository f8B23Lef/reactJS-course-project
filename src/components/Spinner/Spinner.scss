@import '../../styles/colors.scss';

.spinner-container {
  display: flex;
  justify-content: center;
  align-items: center;
}

.spinner {
  width: 50px;
  height: 50px;
  border: 10px solid $light-grey;
  border-top: 10px solid $grey;
  border-radius: 50%;
  animation: spinner 1.5s linear infinite;
}

@keyframes spinner {
  0% {
    transform: rotate(0deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
