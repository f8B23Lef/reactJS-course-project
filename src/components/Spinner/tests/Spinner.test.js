import React from 'react';
import { render } from '@testing-library/react';
import Spinner from '../Spinner';

describe('Spinner', () => {
  it('renders Spinner snapshot', () => {
    const { asFragment } = render(<Spinner />);
    expect(asFragment(<Spinner />)).toMatchSnapshot();
  });
});
