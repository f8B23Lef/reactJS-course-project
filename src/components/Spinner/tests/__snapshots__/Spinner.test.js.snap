// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders Spinner snapshot 1`] = `
<DocumentFragment>
  <div
    class="spinner-container"
  >
    <div
      class="spinner"
    />
  </div>
</DocumentFragment>
`;
